src/ignitus-Routes/notFound.js

```diff
@@ -1,20 +1,37 @@
-
 import React from 'react';
 import './Styles/style.scss';
 import { Link } from 'react-router-dom';
-
-export const Notfound = () => (
-  <div className="container-404">
-    <div className="content">
-      <p>404</p>
-      <p>We could not find what you were looking for.</p>
-      <p>
-		Please check your URL or return&nbsp;
-        <Link to="/">
+import Lottie from 'react-lottie'
```

Comment:
You are missing semi-colon here and please keep in mind this message https://github.com/Ignitus/Ignitus-client/pull/600/files#r390328751. Absolute imports should come before relative.